src/gsi/cplr_get_wrf_mass_ensperts.f90

```diff
@@ -295,6 +306,18 @@ subroutine get_wrf_mass_ensperts_wrf(this,en_perts,nelen,ps_bar)
                   do k=1,grd_ens%nsig
                      do i=1,grd_ens%lon2
                         do j=1,grd_ens%lat2
+                           if (l_use_cvpqx) then ! CV transform
+                               if (qr(j,i,k) <= 1.0E-5_r_kind) then  !Originally Gang used 5.0E-5
+                                  qr(j,i,k) = 1.0E-5_r_kind         !Rong Kong
+                               end if
+                               if (l_use_cvpqx_pval .gt. 0.0_r_kind ) then ! CVpq
+                                  if(mype==0 .and. i==10 .and. j==10 .and. k==10) write(6,*)'power transform for qr : from member-->',n
```

Comment:
see my previous comment on this kind of write command.

Reply:
I modified the code according to your comments. I took out these codes from loops and make them print one time at the start of the routine using if(mype==0).

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-f4844e36d71565749a9ceb0a3aa45ace24257a80b593048fbc2b5585e0bbca8fR353-R369